app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+  protect_from_forgery
+  skip_before_action :verify_authenticity_token
+
+  def index
+    render json:  @users = User.all
```

Comment:
Why are you rendering a json? We are not building an API here, we are building a full rails app.